protected/humhub/modules/admin/views/space/index.php

```diff
@@ -64,6 +64,7 @@
                 'userAttribute' => 'ownerUser',
                 'label' => Yii::t('SpaceModule.base', 'Owner')
             ],
+            ['attribute' => 'sort_order'],
```

Comment:
Not sure we this information here. I would like to keep this overview very compact. Maybe in the future we can allow a view with flexible columns.

Reply:
I think we need to have a place where we can see all spaces order, otherwise it will be too complicated to sort spaces.
In the past, we had the same problem with profile fields, and since we added this column on `/admin/user-profile` it's much better.
This is a view with the Enterprise theme (which reduces width with the left panel):
![image](https://user-images.githubusercontent.com/23310825/235342544-fcf54a24-2a4e-4b74-aedb-5b2bf84fbaa1.png)
Perhaps we could remove the owner names and add them as a tooltip to use this space for sort order?
Ideally, we would have a drag and drop system to sort them, and in this case we don't need anymore to display the sort number.

Reply:
How about for now showing the sort order at the description or after the name "Welcome Space (120)" in brackets if its not the default value.  Regarding the general design/features of the tables (Admin user, Admin spaces) I've created a design tasks.

Reply:
I think few people will understand that this number is the sort order, but perhaps adding "Sort order: " in front of the number could make it clearer.
What do you think about a label floating right?

![image](https://user-images.githubusercontent.com/23310825/235680461-2a04b165-1044-4830-bd37-dd0894f8a8a4.png)

![image](https://user-images.githubusercontent.com/23310825/235680571-9c2563da-9546-4b79-a9d3-6764f4d1b75f.png)

Reply:
Or a lighter solution, adding " / Sort order" to the column label?
![image](https://user-images.githubusercontent.com/23310825/235681367-b07c56bf-ee58-4f3e-a527-5413dcfa8a1e.png)

Reply:
https://github.com/humhub/humhub/commit/30ac4af210b101daadb51d83e815c2c0c3a66d5a

![image](https://user-images.githubusercontent.com/23310825/236155862-753ca444-1920-419e-beb1-d7bff503496c.png)

If no custom sort order, I don't add the `/ Sort order` to the title as I think we wouldn't understand:
![image](https://user-images.githubusercontent.com/23310825/236156163-4f006d18-735b-40ed-9467-c743a96b1e3c.png)

Reply:
@funkycram Nice!